### PR TITLE
myStrom (Independent Publisher)

### PR DESCRIPTION
**Pull request by tposzytek**
Hi! I would like to submit my independent publisher connector for myStrom API. I confirm I have checked, validated, commented the connector inline with the requirements. Also, please find screenshots confirming successful tests of the connector:

[swagger file validation outcome](https://poszytek.eu/wp-content/uploads/2021/09/mystrom-successful-validation.png)
[operations tested successfully](https://poszytek.eu/wp-content/uploads/2021/09/test-operations-section.png)
[Powr Automate run successful](https://poszytek.eu/wp-content/uploads/2021/09/flow-run-successfully.png)

Looking forward to hearing from you, regards
Tomasz

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [X] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [X] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: independent-publisher-connectors/myStrom/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,640 @@
...
+                  "type": "string",
+                  "description": "Token to be used to authenticate other requests.",
+                  "title": "Authentication token"
+                }
+              }
+            }
+          },
+          "default": {
+            "description": "Default response",
+            "schema": {}
```
Comment (sriyen-msft) [inline independent-publisher-connectors/myStrom/apiDefinition.swagger.json:53]: @tposzytek Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid response schema definition for all the operations in the swagger.

File: certified-connectors/Asite (US Gov.)/Readme.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# About
```
Comment (sriyen-msft) [inline certified-connectors/Asite (US Gov.)/Readme.md:1]: @tposzytek Please remove other connector files from your PR

File: independent-publisher-connectors/myStrom/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,640 @@
...
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Response",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "string",
+                  "description": "",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/myStrom/apiDefinition.swagger.json:178]: @tposzytek Empty description values can be removed until unless you have specific reason

File: independent-publisher-connectors/myStrom/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,640 @@
...
+            "description": "The color we set the bulb to. HSV is of form <UINT 0..360>;<UINT 0..100>;<UINT 0..100>",
+            "x-ms-summary": "Color",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "ramp",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "description": "Transition time from the light\u2019s current state to the new state. [ms]",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/myStrom/apiDefinition.swagger.json:312]: @tposzytek Unicodes - The connector's swagger still have unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in the English language.

File: independent-publisher-connectors/myStrom/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,640 @@
...
+            "type": "string",
+            "description": "Authentication Token",
+            "x-ms-summary": "Authentication Token",
+            "format": "password",
+            "x-ms-visibility": "important"
+          }
+        ]
+      }
+    },
+    "/device": {},
```
Comment (sriyen-msft) [inline independent-publisher-connectors/myStrom/apiDefinition.swagger.json:353]: @tposzytek Empty operation definations can be removed as they would not add any value to the connector.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/myStrom/apiDefinition.swagger.json (hunk 1 above)
Comment (tposzytek) [reply independent-publisher-connectors/myStrom/apiDefinition.swagger.json:53]: Done

File: independent-publisher-connectors/myStrom/apiDefinition.swagger.json (hunk 3 above)
Comment (tposzytek) [reply independent-publisher-connectors/myStrom/apiDefinition.swagger.json:178]: Done

File: independent-publisher-connectors/myStrom/apiDefinition.swagger.json (hunk 4 above)
Comment (tposzytek) [reply independent-publisher-connectors/myStrom/apiDefinition.swagger.json:312]: Done

File: independent-publisher-connectors/myStrom/apiDefinition.swagger.json (hunk 5 above)
Comment (tposzytek) [reply independent-publisher-connectors/myStrom/apiDefinition.swagger.json:353]: Done

**Comment by tposzytek** [comment]
Changes

**Pull request closed without merging**